portality/view/api_v4.py

```diff
@@ -0,0 +1,98 @@
+import json
+import re
+
+from flask import Blueprint, request
+from flask import jsonify, url_for, make_response, render_template
+from flask_login import current_user
+from flask_swagger import swagger
+
+from portality.api import respond
+from portality.api.common import Api400Error
+from portality.api.current import ArticlesBulkApi
+from portality.core import app
+from portality.decorators import api_key_required, swag, write_required
+from portality.lib import plausible
+from portality.models import BulkArticles
+from portality.view import api_v3
```

Comment:
I think we should be doing this the other way around - the api_v3 should forward compatible requests to api_v4, rather than the other way around.  This means that the full implementation of the current API is present in the most up-to-date version file, and the old api implementations are just forwarders.  

So in api_v3, I would import api_v4 and send on the requests from there to here, rather than importing api_v3 here and sending requests back to it.

Reply:
Also api_v3 appears to import api_v4, so isn't this a circular import?

Reply:
You can see examples of how this is done in api_v1 and api_v2, I'd like api_v3 to behave the same.